composer.json

```diff
@@ -26,6 +26,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
+        "ext-curl": "*",
```

Comment:
This one is an optional dependency. It's only needed if you are using the .NET SOAP client implementation. As such, it should be in the `"suggest"` section instead.